### PR TITLE
Outline of a Skrifa fuzzer

### PR DESCRIPTION
**Pull request by rsheeter**
First step toward #420. Fuzzer is currently nop. Meant to enable oss-fuzz setup and establish a pattern we can use to add additional fuzzers for subsetting, etc.

For context, first swing at oss-fuzz rigging in https://github.com/google/oss-fuzz/pull/12011.

JMM

### CODE REVIEW

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[package]
+name = "fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
```
Comment (cmyr) [inline fuzz/Cargo.toml:6]: I think we might also want,

```toml
# cargo-release settings
[package.metadata.release]
release = false
```

**Review by cmyr — APPROVED** [review]
one note, otherwise lgtm

File: fuzz/Cargo.toml (hunk 1 above)
Comment (rsheeter) [reply fuzz/Cargo.toml:6]: Will add, ty

Comment (rsheeter) [reply fuzz/Cargo.toml:6]: Fixed, ty

**Pull request merged**